js/main.js

```diff
@@ -17,6 +17,60 @@ const getRandomFloatInt = (min, max, precision) => {
   throw new Error('getRandomFloatInt - диапазон или точность не соответствуют условию');
 };
 
-getRandomRoundedInt(1, 3);
-getRandomFloatInt(1.2, 1.3, 2);
+const TOTAL_USERS = 8;
+const PRICE = {
+  min: 5000,
+  max: 100000,
+};
+const TYPE = ['palace', 'flat', 'house', 'bungalow'];
+const ROOMS = {
+  min: 1,
+  max: 6,
+};
+const GUESTS = {
+  min: 1,
+  max: 12,
+};
+const CHECKIN_TIME = ['12:00', '13:00', '14:00'];
+const CHECKOUT_TIME = ['12:00', '13:00', '14:00'];
+const FEATURES = ['wifi', 'dishwasher', 'parking', 'washer', 'elevator', 'conditioner'];
```

Comment:
Писал тебе о нейминге в переписке:
стоит именовать константы более конкретно. Мало ли о каких фичах речь 
```suggestion
const ROOM_FEATURES = ['wifi', 'dishwasher', 'parking', 'washer', 'elevator', 'conditioner'];
```
Аналогично с другими константами